src/components/SingleName/NameRegister/Calendar.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import AddToCalendar from 'react-add-to-calendar'
+
+function CalendarInvite(
```

Comment:
Where do I see the UI in action? I didn't see calendar feature while registering

Reply:
There isn't atm. I built the calendar functionality, but realised we only need it for the auction